### ISSUE TITLE
Sample data-label file and model prototxt

### ISSUE THREAD

**Issue by ashwin-999** [tyandzx/caffe#7]
Hi,

I'm trying to use your triplet loss for cifar10. However my train loss isn't decreasing, and my validation accuracy is always 0. I believe the problem is with my input. 

Can you share an example model along with image-label (for certain batch size) file which will help us understand what is required. 

Thank you
